mono/metadata/icall-decl.h

```diff
@@ -179,5 +179,12 @@ ICALL_EXPORT void ves_icall_System_Runtime_RuntimeImports_Memmove (guint8*, guin
 ICALL_EXPORT void ves_icall_System_Runtime_RuntimeImports_Memmove_wbarrier (guint8*, guint8*, guint, MonoType*);
 ICALL_EXPORT void ves_icall_System_Runtime_RuntimeImports_ZeroMemory (guint8*, guint);
 ICALL_EXPORT void ves_icall_System_Runtime_RuntimeImports_ecvt_s(char*, size_t, double, int, int*, int*);
+#if defined(ENABLE_MONODROID) || defined(ENABLE_MONOTOUCH)
```

Comment:
would be better to have a feature macro, instead of platforms.

Reply:
We are only interested in having the for the `monodroid`/`monotouch{_tv,_watch}` profiles (as per the BCL), so the feature macro is kinda already `ENABLE_MONODROID`/`ENABLE_MONOTOUCH`.